src/pint/utils.py

```diff
@@ -1667,3 +1667,17 @@ def remove_dummy_distance(c):
             "Do not know coordinate frame for %r: returning coordinates unchanged" % c
         )
         return c
+
+
+def info_string(prefix_string="# ", comment=None):
+    import pint
+    import getpass
+    import platform
+    import datetime
```

Comment:
Is there a reason these should be local imports rather than at the top of the file as normal?

Reply:
No, that's probably just a result of me testing and then not properly integrating. fixed.